test/src/test/java/hudson/PluginManagerTest2.java

```diff
@@ -1,48 +0,0 @@
-package hudson;
-
-import java.io.File;
-import java.io.IOException;
-import java.net.URL;
-
-import javax.servlet.ServletContext;
-
-import org.jvnet.hudson.test.HudsonTestCase;
-import org.xml.sax.SAXException;
-
-import com.gargoylesoftware.htmlunit.html.HtmlForm;
-import com.gargoylesoftware.htmlunit.html.HtmlPage;
-import org.apache.commons.io.FileUtils;
-
-public class PluginManagerTest2 extends HudsonTestCase {
```

Comment:
@tfennelly deleted the only test case in https://github.com/jenkinsci/jenkins/pull/1995 but neglected to delete the suite as well.